en/index.html

```diff
@@ -240,7 +240,30 @@
 							</li>
 							<li><strong>OS X</strong>
 								<ul>
-									<li>I have never used OS X, so don't ask me :(</li>
+									<li>
+										Some would recommend NetBeans, but I wouldn't want you to waste time doing all
+										the setup stuff. Believe me, setup for a program always makes anything
+										interesting into boring.
+									</li>
+									<li>
+										If you have heard of PhpStorm, again, same with NetBeans, setup trouble
+										(although there isn't a lot). Moreover, it is paid software, although there are
+										some ways to legitimately get it for free. Nevertheless, it is an all-powerful
+										IDE that completes the code for you when you have provided a little hint, and
+										can also be used for other things like coding websites, etc. Actually, this
+										website was written using PhpStorm (although there is no PHP here).
+									</li>
+									<li><strong>Browser + GitHub</strong>
+										The GitHub editor for code is not bad, at least better than Windows Notepad, but
+										it is far not as good as PhpStorm. If you don't like installing anything, this
+										is a good alternative to Notepad++.
+									</li>
+									<li><strong>TextMate</strong>
+										<em><a
```

Comment:
Because i wanted to keep your code style
https://github.com/PEMapModder/PocketMine-Plugin-Tutorials/blob/gh-pages/en/index.html#L217